src/pages/CreatePool/index.tsx

```diff
@@ -122,16 +130,11 @@ export default function CreatePool(): React.ReactElement {
                   label={`${t("fee")} (%)`}
                   fullWidth
                   value={fee}
-                  error={
-                    !digitRegex || parseFloat(fee) > 1 || parseFloat(fee) < 0.04
-                  }
+                  type="text"
+                  inputProps={{ inputMode: "numeric", pattern: "[0-9]*" }}
```

Comment:
I don't think this prop does anything after experimenting with it. Does it do something?

Reply:
Actually this props works after user click submit button on the form.
Because we disabled `create pool` button until every field is filled and don't have error, so we don't need this props.